src/lib/confSchema.ts

```diff
@@ -0,0 +1,77 @@
+import type { FromSchema } from "json-schema-to-ts";
+import { JSONConfiguration } from "./JSONConfiguration";
+
+export const exampleSchema = {
```

Comment:
can we make it type safe?

Reply:
Yeah, that is the issue I mentioned ealier:
- this object can be typesafe by using `import { RJSFSchema } from "@rjsf/utils";` type 
- but using this type breaks `json-schema-to-ts`, which can produce a type of defined schema